src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -44,7 +44,8 @@ public class ConsoleRunner
         public static readonly int OK = 0;
         public static readonly int INVALID_ARG = -1;
         public static readonly int INVALID_ASSEMBLY = -2;
-        public static readonly int FIXTURE_NOT_FOUND = -3;
+        //public static readonly int FIXTURE_NOT_FOUND = -3;    //No longer in use
+        public static readonly int INVALID_TEST_FIXTURE = -4;
```

Comment:
Commented out the FIXTURE_NOT_FOUND code here and in NUnit lite as it's no longer in use. Left it in rather than removing it entirely so it's not accidentally repurposed.